docker/compose/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "2"
+services:
+  jagger-mysql:
+    image: mysql:5.7
+    container_name: jagger-mysql
+    # Expose ports. Either specify both ports (HOST:CONTAINER), or just the
+    # container port (a random host port will be chosen).
+    ports:
+      - 3306:3306
+    # Set mysql environments
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_USER: jaggeruser
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: jaggerdb  
+    # Copy mysql config to the container
+    volumes:
+      - ./files/my.cnf:/etc/mysql/conf.d/my.cnf
```

Comment:
I don't know weather it will create new directory or not in case there is no such one in your local OS. It may ask for additional rights from user.